README.md

```diff
@@ -402,6 +403,13 @@ ESLint rules:
 - [`space-infix-ops`](https://eslint.org/docs/rules/space-infix-ops.html)
 - [`comma-spacing`](http://eslint.org/docs/rules/comma-spacing)
 - [`no-multiple-empty-lines`](http://eslint.org/docs/rules/no-multiple-empty-lines)
+- [`max-len`](http://eslint.org/docs/rules/max-len)
+- [`max-lines-per-function`](http://eslint.org/docs/rules/max-lines-per-function)
+
+SonarQube rules:
+
+- `Lines should not be too long`
+- [`Functions should not too many lines of code`](https://rules.sonarsource.com/javascript/RSPEC-138?search=Functions%20should%20not%20too%20many%20lines%20of%20code)
```

Comment:
Typo: Functions should not **have** too many lines of code

Reply:
fixed :)